nixos/modules/services/networking/syncthing.nix

```diff
@@ -126,11 +126,16 @@ let
     builtins.attrValues
     (lib.concatStringsSep "\n")
   ]) +
+  /* Now we hash the contents of guiPasswordFile and use the result to update the gui password */
+  (lib.optionalString (cfg.guiPasswordFile != null) ''
+     ${pkgs.mkpasswd}/bin/mkpasswd -m bcrypt --stdin <"${cfg.guiPasswordFile}" | tr -d "\n" >$RUNTIME_DIRECTORY/password_bcrypt
```

Comment:
I think it's good style to quote everything that would break if it contains space (although in this context it's unlikely).